.circleci/config.yml

```diff
@@ -421,3 +421,17 @@ workflows:
           <<: *build_on_tags
           requires:
             - build_x86_archlinux
+  test_nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
```

Comment:
Is that GMT?

Reply:
It's UTC. So this would be executed at 1:00 / 2:00 AM CET, if I'm not mistaken.